### ISSUE TITLE
Could not determine repo and owner

### ISSUE THREAD

**Issue by fabiocaccamo** [codecov/codecov-bash#465]
I switched from Travis CI to GitHub actions (using `tox-gh-actions`) for testing my django packages.

I'm using the same tox file, but now the coverage report is not uploaded to codecov anymore, as you can see [here](https://app.codecov.io/gh/fabiocaccamo/django-colorfield) there are no more reports uploaded since September 2021.

Looking inside the [GitHub Actions job](https://github.com/fabiocaccamo/django-colorfield/actions/runs/1612327892) the error is: **Could not determine repo and owner**

<img width="715" alt="Screenshot 2021-12-22 at 22 47 48" src="https://user-images.githubusercontent.com/1035294/147158920-d34830d5-232c-4293-8908-d7697f3933e6.png">

Is there a way to avoid/fix this?

**Comment by DuncanTormey** [comment]
@fabiocaccamo did you find a solution? I am seeing the same issue.

**Comment by fabiocaccamo** [comment]
@DuncanTormey yes, I stopped using `tox-gh-action` and created this GitHub action that allows you to use a tox-like matrix definition directly in the GH workflow:

https://github.com/fabiocaccamo/create-matrix-action
